PowerQueryM/ForecastedLoad.query.pq

```diff
@@ -0,0 +1,32 @@
+let
+    MCastDomain = "demo-gas.mea-analytics.tools",
+    MaxDate = DateTime.Date(DateTime.LocalNow()),
+    MinDate = Date.AddDays(MaxDate, -60),
+    ApiKey = "your API Key here",
+    Headers = [#"x-api-key"=ApiKey],
+    Options = [Headers=Headers],
+
+    ForecastHorizon = 1,
+    ForecastQueryParams = [ 
+        operatingArea = "Metropolis",
```

Comment:
Hmmm there are also things that need editing in here. I'm just wondering what is the best way is to group all the variables that we know need changing.

Reply:
I'll play around with it